<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>273</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
	width:40px;
	height:40px;
	border:2px solid black;
	border-radius:10px;
}
QPushButton:pressed{
	background:gray;
	border:1px solid black;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>70</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>19</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel#label{
	background:white;
	font-size:40;
	text-align:right;
	qproperty-alignment:'AlignRight|AlignVCenter';
	height:20px;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_Del">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Del</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_AC">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>AC</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_per">
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background:rgb(245, 121, 0);
}</string>
          </property>
          <property name="text">
           <string>%</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_divide">
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background:rgb(245, 121, 0);
}</string>
          </property>
          <property name="text">
           <string>/</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton_7">
          <property name="text">
           <string>7</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_8">
          <property name="styleSheet">
           <string notr="true">QPushButton{
	width:40px;
	height:40px;
}</string>
          </property>
          <property name="text">
           <string>8</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_9">
          <property name="text">
           <string>9</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_multiply">
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background:rgb(245, 121, 0);
}</string>
          </property>
          <property name="text">
           <string>*</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_5">
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_6">
          <property name="text">
           <string>6</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_minus">
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background:rgb(245, 121, 0);
}</string>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="pushButton_1">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_plus">
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background:rgb(245, 121, 0);
}</string>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="pushButton_0">
          <property name="maximumSize">
           <size>
            <width>16777100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	width:40%;
}</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_dec">
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>42</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>17</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_calc">
          <property name="minimumSize">
           <size>
            <width>118</width>
            <height>48</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>35</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>=</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
